horaedb/metric_engine/src/manifest.rs

```diff
@@ -69,8 +82,29 @@ impl From<Payload> for pb_types::Manifest {
 }
 
 impl Manifest {
-    pub async fn try_new(path: String, store: ObjectStoreRef) -> Result<Self> {
+    pub async fn try_new(
+        path: String,
+        store: ObjectStoreRef,
+        merge_options: SstsMergeOptions,
+    ) -> Result<Self> {
         let snapshot_path = Path::from(format!("{path}/{SNAPSHOT_FILENAME}"));
+        let sst_path = Path::from(format!("{path}/{SST_PREFIX}"));
+
+        let (tx, rx) = mpsc::channel(merge_options.channel_size);
+        let mut ssts_merge = SstsMerge::try_new(
+            snapshot_path.clone(),
+            sst_path.clone(),
+            store.clone(),
+            rx,
+            merge_options,
+        )
+        .await?;
+
+        // Start merge ssts task
+        tokio::spawn(async move {
+            ssts_merge.run().await;
+        });
+
```

Comment:
When start up, we also need to read delta files, if we only read snapshot, then part of sst won't be queried later?